src/Motion.js

```diff
@@ -42,22 +49,22 @@ const Motion = createReactClass({
       lastIdealStyle: currentStyle,
       lastIdealVelocity: currentVelocity,
     };
-  },
+  }
 
-  wasAnimating: false,
-  animationID: (null: ?number),
-  prevTime: 0,
-  accumulatedTime: 0,
+  wasAnimating: false;
```

Comment:
Need to update types to include it, I reckon.

I am not doing much of Flow and never used it with JS Classes, so cannot help much in this case. Maybe someone can give you better advice.

Or there must be some examples of using Flow with React Classes somewhere